Drivers/Periphs/Inc/IMU.h

```diff
@@ -19,6 +19,185 @@
 
 #include "FreeRTOS.h"
 #include <stdint.h>
+#include "stm32l4xx_hal.h"
+#include "spi.h"
+
+#define COMMAND_ARR_SIZE 10
+
+#define LSM6_SPI_READ 0b1
+#define LSM6_SPI_WRITE 0b0
+
+/*LSM6 Registers*/
+#define LSM6_CFG_ACCESS 0x01
```

Comment:
any reason why these are all `#define`s? the header I gave ya'll uses enums to make things a little more clear and also gives enums for odr settings